### ISSUE TITLE
[Bug] Scrollbar is scrolled when cursor hovers over it

### ISSUE THREAD

**Issue by hyuri** [story-apps/starc#117]
When I hover over the scrollbar, it widens and scrolls.

Seems related to #115 

[Hovering Over Scrollbar Causes Scrolls - Screencast.webm](https://github.com/story-apps/starc/assets/18584014/cc3b4bb6-3b5d-45d7-9825-c1d10f87ff97)

#### App Info
Starc v0.5.2 (Flatpak)

#### System Info
Pop!_OS 22.04
X11

**Comment by dimkanovikov** [comment]
yup, it should be automatically fixed when we find a way to implement #115
